packages/fast-element/src/directives/directive.ts

```diff
@@ -19,7 +19,7 @@ export class AttachedBehaviorDirective<T = any> extends Directive {
     }
 
     public createPlaceholder(index: number) {
-        return `${this.name}="${DOM.createInterpolationPlaceholder(index)}"`;
+        return DOM.createCustomAttributePlaceholder(this.name, index);
```

Comment:
This change is unrelated to the main purpose of this PR. I just moved this code into the `DOM` helper in order to facilitate reuse and participate in the DOM's frozen characteristic.